### DISCUSSION TITLE
Sort by or order by in REST API

### DISCUSSION THREAD

**Discussion by Brizolara84** [supabase/supabase#13750] [Questions]
Guys, do we have any way in GET to bring the orderer records?

This Error 

{
  "code": "PGRST104",
  "details": "Failed to parse [(\"sort\",\"nrorder\")]",
  "hint": null,
  "message": "Unexpected param or filter missing operator"
}

**Comment by GaryAustin1** [comment]
You don't provide much information on what code you are running.
The REST API Supabase uses is completely defined here https://postgrest.org/en/stable/api.html
You can order records in the API.
Looking at your details: above it does not make any sense as to what the `[("sort","norder")]` is.

**Reply by Brizolara84** [reply]
Hi Gary,

I was wanting to sort the return data, I saw that some API's support sorting the data through some field

**Reply by GaryAustin1** [reply]
The link I sent before shows how to do sorts for the REST API.
https://postgrest.org/en/stable/api.html#ordering
